playbooks/byo/openshift-cluster/upgrades/docker/docker_upgrade.yml

```diff
@@ -23,11 +23,23 @@
   serial: 1
   any_errors_fatal: true
   tasks:
-  - name: Prepare for Node draining
-    command: >
-      {{ openshift.common.client_binary }} adm manage-node {{ openshift.node.nodename }} --schedulable=false
+  - name: Load lib_openshift modules
+    include_role:
+      name: lib_openshift
```

Comment:
I guess in this case I can add a `roles:` section since that will be run before the tasks.  Will update.